DataUsageAgreement.md

```diff
@@ -0,0 +1,27 @@
+Texas Justice Initiative     
+P.O. Box 164286  
+Austin, TX 78716   
+https://www.texasjusticeinitiative.org  
+
+![](https://raw.githubusercontent.com/texas-justice-initiative/website-nextjs/master/images/tji-logo.svg) 
```

Comment:
```suggestion
<img src="https://raw.githubusercontent.com/texas-justice-initiative/website-nextjs/master/images/tji-logo.svg" alt="TJI logo" style="max-width:200px;">
```

We could do something like this if we wanted to make the logo more reasonably sized in the rendered version.